trufflescripts/start_auction.js

```diff
@@ -19,6 +19,8 @@ module.exports = async () => {
 
   // auctionStart is in the future
   if (timeUntilStart > 0) {
+    const snapshot = await makeSnapshot()
+    console.log(`SNAPSHOT CREATED OF BLOCK # ${snapshot}`)
```

Comment:
I don't think snapshot corresponds to block #
It's just an incrementing id.
`makeSnapshot` is sync, no need to await, same with `revertSnapshot` above